src/scripts/heroAnimation/hero.js

```diff
@@ -0,0 +1,256 @@
+}
+
+function scrollCheck() {
+  if (document.body.scrollTop > height) {
+    animateHeader = false;
+  } else {
+    animateHeader = true;
+  }
+}
+
+function resizeCanvas() {
+  canvasTop = canvas.getBoundingClientRect().top;
+  canvas.setAttribute(
+    "style",
+    `width: ${container.clientWidth}px; height: ${container.clientHeight}px;`
+  );
+}
+
+function drawLines(p) {
+  if (!p.active) return;
+  for (var i in p.closest) {
+    ctx.beginPath();
+    ctx.moveTo(p.x, p.y);
+    ctx.lineTo(p.closest[i].x, p.closest[i].y);
+    ctx.strokeStyle = "rgba(94, 161, 184," + p.active + ")";
+    ctx.stroke();
+  }
+}
+
+function Circle(pos, rad, color) {
+  var _this = this;
+
+  // constructor
+  (function() {
+    _this.pos = pos || null;
+    _this.radius = rad || null;
+    _this.color = color || null;
+  })();
+
+  this.draw = function() {
+    if (!_this.active) return;
+    ctx.beginPath();
+    ctx.arc(_this.pos.x, _this.pos.y, _this.radius, 0, 2 * Math.PI, false);
+    ctx.fillStyle = "rgba(94, 161, 184," + _this.active + ")";
+    ctx.fill();
+  };
+}
+
+function animate() {
+  if (animateHeader) {
+    ctx.clearRect(0, 0, width, height);
+    for (var i in points) {
+      // detect points in range
+      if (Math.abs(getDistance(target, points[i])) < 8000) {
+        points[i].active = 0.3;
+        points[i].circle.active = 0.75;
+      } else if (Math.abs(getDistance(target, points[i])) < 40000) {
+        points[i].active = 0.1;
+        points[i].circle.active = 0.3;
+      } else if (Math.abs(getDistance(target, points[i])) < 80000) {
+        points[i].active = 0.04;
+        points[i].circle.active = 0.1;
+      } else {
+        points[i].active = 0.06;
+        points[i].circle.active = 0.06;
+      }
+
+      drawLines(points[i]);
+      points[i].circle.draw();
+    }
+  }
+  requestAnimationFrame(animate);
+}
+
+function shiftPoint(p) {
+  TweenLite.to(p, 1 + 1 * Math.random(), {
+    x: p.originX - 0 + Math.random() * 20,
+    y: p.originY - 0 + Math.random() * 20,
+    ease: Circ.easeInOut,
+    onComplete: function() {
+      shiftPoint(p);
+    }
+  });
+}
+
+function mouseMove(e) {
+  let posy = 0;
+  let posx = (posy = 0);
+
+  posx = e.pageX;
+  posy = e.pageY - canvasTop;
+
+  target.x = posx;
+  target.y = posy;
+}
+
+function mouseEnter() {
+  initAnimation();
+}
+
+function mouseOut() {
+  target.x = container.clientHeight * 2;
+  target.y = container.clientHeight * 2;
+  resetTimer();
+}
+
+function getDistance(p1, p2) {
+  return Math.pow(p1.x - p2.x, 2) + Math.pow(p1.y - p2.y, 2);
+}
+
+function addListeners() {
+  let timer = null;
+
+  if (!("ontouchstart" in window)) {
+    container.addEventListener("mousemove", mouseMove);
+    container.addEventListener("mouseout", mouseOut);
+    container.addEventListener("mouseenter", mouseEnter);
+    window.addEventListener("resize", resizeCanvas);
+  }
+
+  window.addEventListener("scroll", scrollCheck);
+
+  window.onresize = function() {
```

Comment:
I think this was meant to be removed, no?

As it stands, all viewport resizes (including mobile browsers hiding their address bars) trigger full-page reloads.